app/models/tag.rb

```diff
@@ -1,4 +1,9 @@
 class Tag < ApplicationRecord
   has_many :jobs_tags
   has_many :jobs, through: :jobs_tags
+
+  def jobs_with_same_tag
+    Tag.where(name: self.name).count
+  end
```

Comment:
Is there something we could implement to avoid creating multiple tags of the same name?